code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -58,6 +58,7 @@
 	var/radiation_mod = 1                    // Radiation modifier
 	var/flash_mod =     1                    // Stun from blindness modifier.
 	var/vision_flags = SEE_SELF              // Same flags as glasses.
+	var/see_infrared = FALSE
```

Comment:
please leave a commit on what this means

Reply:
Done